### PR TITLE
Add vote notify permission

### PR DESCRIPTION
**Pull request by Pixelhash**
Hi,

I think the title says everything.

### CODE REVIEW

File: src/main/java/io/minimum/minecraft/superbvote/votes/SuperbVoteListener.java [hunk 1]
```diff
@@ -104,7 +104,8 @@ public void onPlayerJoin(PlayerJoinEvent event) {
             votes.forEach(v -> processVote(v, false, false, true));
 
             // Remind players to vote.
-            if (SuperbVote.getPlugin().getConfig().getBoolean("vote-reminder.on-join")) {
+            if (SuperbVote.getPlugin().getConfig().getBoolean("vote-reminder.on-join") &&
+                    event.getPlayer().hasPermission("superbvote.notify")) {
```
Comment (minecrafter) [inline src/main/java/io/minimum/minecraft/superbvote/votes/SuperbVoteListener.java]: Without a default permission setting, notifications will only be sent to ops.

File: src/main/java/io/minimum/minecraft/superbvote/votes/VoteReminder.java [hunk 2]
```diff
@@ -8,7 +8,9 @@
     @Override
     public void run() {
         for (Player player : Bukkit.getOnlinePlayers()) {
-            SuperbVote.getPlugin().getConfiguration().getReminderMessage().sendAsReminder(player);
+            if (player.hasPermission("superbvote.notify")) {
```
Comment (minecrafter) [inline src/main/java/io/minimum/minecraft/superbvote/votes/VoteReminder.java]: Without a default permission setting, notifications will only be sent to ops.

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+# Maven
+/target
+dependency-reduced-pom.xml
+
+# IntelliJ
+*.iml
+*.ipr
+*.iws
+.idea/
```
Comment (minecrafter) [inline .gitignore]: Don't mix unrelated changes.

File: pom.xml [hunk 4]
```diff
@@ -75,7 +75,7 @@
         <dependency>
             <groupId>org.spigotmc</groupId>
             <artifactId>spigot-api</artifactId>
-            <version>1.8.8-R0.1-SNAPSHOT</version>
+            <version>1.11.2-R0.1-SNAPSHOT</version>
```
Comment (minecrafter) [inline pom.xml]: Unrelated change.

**Comment by Pixelhash** [comment]
Hi again,

I changed all as requested.
